charters/DataPortal/charter.md

```diff
@@ -0,0 +1,81 @@
+* DevSecOps features for FISMA moderate deployments (authentication and authorization, logging, auditing, etc.) by leveraging other projects (if this becomes a future requirement)
+* Community engagement
+   * Triage and integration of feature requests into the work queue 
+   * Support plan for escalation of user reported bugs
+### Data Portal
+* Select and deploy a content management system
+* Integrate Data Browser and Data Portal UIs into a seamless UX and provide link to other groups' applications (e.g. a Data Ingestion UI)
+* Frequent updates of summary statistics (cells, organs, donors, projects, labs, etc.) on the Home Page
+* Coordinate development of user guide content (e.g. “Learn”) from multiple contributors
+* Coordinate development of developer documentation content (e.g. “Develop”) from multiple contributors. Includes API documentation and development guides.
+* Click through to Data Browser from search box and human body image
+* Registration form for tertiary portals and list of portals available with click through to hosting web site (e.g. Dockstore)
+* Integration with Zendesk help desk
+* Integration with Vanilla Forums (discussion forum for developers)
+* Integration with Canny (to capture user feedback)
+### Data Browser
+* Create (or use Data Storage Service [DSS] provided) indexes to access files in the Data Store by using public DSS APIs to read the metadata
+* Browser support for multiple metadata schemas that are supported by the schema access library maintained by the Ingest service
+* Search views of Specimens, Projects and Files
+* Browser UI to search the DSS by specifying multiple search criteria (“facets”)
+* Download selected files from the DSS via the UI and/or HCA cli
+* Full authentication and authorization if the HCA stores controlled access data in the future. Leverage the authetication and authorization implementation from the Data Store.
+* Browser interface to the DSS Collections service to save, view, list, share and delete the query results from a search operation
+* Handoff of a Collection of search results to a tertiary portal. Provide for direct handoff of search results to FireCloud.
+* Facilities to save, view, share, update, and delete search queries and personal collections on a per-user basis
+* Search results handoff to the Matrix service
+* API Documentation for portable, cloud-neutral APIs
+
+## Out-of-scope
+* Production operation of other HCA DCP components in AWS (e.g. Data Store Service, data ingestion process, Secondary Analysis workflows, Tertiary Analysis portals, etc.)
+* Any separate data ingest application developed by The European Bioinformatics Insitute (EBI) which may replace or augment the "Contribute" tab in the Data Portal
+* Methods Registry providing management, searching, filtering, sorting, visualization, and methods packages in the tertiary portal (registry is developed and maintained by the Broad Institute)
+* The HCA cli implementation
```

Comment:
Please use _HCA DCP Command Line Interface (CLI)_ and what specifically is out-of-scope - the infrastructure implementation that's part of the Data Store charter?

Reply:
yes - done.